website/docs/Installation/linux-install/index.md

```diff
@@ -21,7 +21,7 @@ Alternatively, you can install Podman Desktop from:
 #### Prerequisites
 
 - [Flatpak](https://flatpak.org/setup/)
-- [Podman](https://podman.io/whatis.html) stable version
+- [Podman](https://docs.podman.io/) stable version
```

Comment:
what about https://podman.io/docs/installation#installing-on-linux ?

```suggestion
- [Podman](https://podman.io/docs/installation#installing-on-linux) stable version
```

Reply:
The "Download" button links there anyway, so might as well use https://podman.io/ in that case.